src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -185,6 +185,10 @@ public static void WriteParameter(this CodeWriter writer, Parameter clientParame
                     writer.WriteConstant(clientParameter.DefaultValue.Value);
                 }
             }
+            else if (!clientParameter.IsRequired)
+            {
+                writer.Append($" = null");
```

Comment:
I think we can use `$" = default"` here and merge this condition with the one below